v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/processor/InputRecordProcessor.java

```diff
@@ -147,10 +146,8 @@ public static boolean processRecord(
       lagMetric.update(replicationLag); // update the lag metric
       return false;
     } catch (Exception e) {
-      LOG.error(
-          "The exception while processing shardId: {} is {} ",
-          shardId,
-          ExceptionUtils.getStackTrace(e));
+      // Not logging the error here since the error can be retryable error and high number of them
```

Comment:
please mark this as an info log and retain here. If a bunch of errors are happening, we should see it somewhere.

Reply:
This is a bad idea for the reasons I described above. In general, it is an anti-pattern to log per-event for EMR pipelines.

> If a bunch of errors are happening, we should see it somewhere.

Metrics should indicate that errors (and their quantum) are happening.

Reply:
Resolving the comment as of now since we need this fix urgently. We can come back and discuss about this later.